parm/config/config.fv3

```diff
@@ -42,8 +48,8 @@ case $case_in in
         export layout_y=2
         export layout_x_gfs=3
         export layout_y_gfs=2
-        export nth_fv3=1
-        export nth_fv3_gfs=1
+        export nth_fv3=2
+        export nth_fv3_gfs=2
```

Comment:
Have these values been tested on one of the other R&Ds to check they work there as well? These lines are used on all platforms.

Reply:
@KateFriedman-NOAA Yes, thanks for reminding me.  During comparison testing on Hera, these jobs failed for running out of memory.  You had suggested in an email that I include this fix in this port.  I can separate it to its own PR if desired, though.

Reply:
If changing the C48 values also helps on Hera then I am happy to have them stay in this PR. Thanks for reminding me we discussed that earlier! @aerorahul thoughts?

@DavidHuber-NOAA Do you happen to know if this works and/or helps on Orion as well? Orion has more memory than Hera so it may not matter in terms of memory but would want to know it doesn't fail. Thanks!

Reply:
@KateFriedman-NOAA I believe I tested this on Orion as well, but it was some time ago and I can't find the log files.  That said, I'm going to start a few cycled tests on Hera and Orion to run over the weekend and I will include this.

Reply:
@KateFriedman-NOAA Ah, I now see the `binary_diag` option in config.base.  I will give it another go with that `=".true."`.

Reply:
This fixed the C96 analysis jobs.  The gdaspostanl and gfspostanl jobs are both failing, as reported in #1006.

The gdaseupd job is failing when `binary_diag=".true."`.  I believe this is a separate issue and will report it after checking if it exists at C96 ensemble resolution.  The problem seems to stem from the output from enkf.x not being in the expected binary format.  When run with newer diag files and `netcdf_diag=.true."` at C48 resolution, the job runs correctly.